sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveSessionCatalog.scala

```diff
@@ -230,22 +230,19 @@ class ResolveSessionCatalog(
 
     case d @ DescribeNamespace(SessionCatalogAndNamespace(_, ns), _) =>
       if (ns.length != 1) {
-        throw new AnalysisException(
-          s"The database name is not valid: ${ns.quoted}")
+        throw new AnalysisException(s"The database name is not valid: ${ns.quoted}")
```

Comment:
Shall we revert this irrelevant change?

Reply:
Ur, sorry, but I suggested in my earlier review: https://github.com/apache/spark/pull/27897#discussion_r392563339
Yea, as you said here, better not to touch these lines in this PR. @Eric5553 Plz revert this? Thanks.

Reply:
Sure, I've reverted these untouched lines. Thanks as always @dongjoon-hyun @maropu !